context_windows.go

```diff
@@ -0,0 +1,9 @@
+package continuity
+
+import "os"
+
+// getUidGidFromFileInfo extracts the user and group IDs from a fileinfo.
+// This is Unix-specific functionality.
```

Comment:
Is there still not an owner and group in msft? We made these strings specifically to be compatible here. Seems odd removing it. How will this data be recovered when used with continuity?